pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-inventory-storage</artifactId>
   <groupId>org.folio</groupId>
-  <version>23.1.0-SNAPSHOT</version>
+  <version>23.1.1-SNAPSHOT</version>
```

Comment:
I think this is right.. He updated to whatever is in master. Which is currently on that next version. Later we can decide what PR should actually update the next version. May be this one.. Maybe at release time.

Reply:
> I think this is right.. He updated to whatever is in master. Which is currently on that next version.

Our release docs explicitly state that the version should move to the next minor upon formal release. Given that the minor has already changed on the mainline and no further releases have been made, why increment the patch version at all?

> Later we can decide what PR should actually update the next version. May be this one.. Maybe at release time.

I also disagree with this. 

I've found that deferring the version change on the release tends to have two effects:
* the person releasing doesn't realise that the version needs to change (and so no change occurs)
* there are no snapshot builds for the intended release version

Personally, I think the version should be incremented at the point when the changes are made that justify the version change.

Reply:
Oops.. Forget what I said. the version in this branch should be same as in master - most certainly not with only patch changed.

Reply:
As for when the major/minor is changed.. It goes two ways .. If it's anybody's job, then in effect anybody needs to know if the major/minor is  *already* changed.

Reply:
> It goes two ways .. If it's anybody's job, then in effect anybody needs to know if the major/minor is _already_ changed.

That's true. 

I may be being a selfish lead maintainer, in thinking that it's easier for folks to check if the version has already changed rather than the lead maintainer needing to understand the importance of every change made during the release cycle.

In theory, it should be easy to tell (although it assumes folks follow the release docs). 

If it's on a non-zero minor, then the major hasn't changed. If it's on a zero minor then it has already changed and no need to do it again.

Reply:
If people actually update fix-version for their stuff and also (they should) tell in issue if this is a breaking change as well in the commit message, then it's easier.. But generally, it's good that the release manager has a PO-role. If they don't know anything about what they are releasing and why, then it's a problem regardless.

Reply:
> If people actually update fix-version for their stuff and also (they should) tell in issue if this is a breaking change as well in the commit message, then it's easier.. 

I haven't found that folks in FOLIO do most of those things. Typically commit messages don't tell folks much in the future and issues rarely include a statement about how they affect the modules compatibility.

> But generally, it's good that the release manager has a PO-role. If they don't know anything about what they are releasing and why, then it's a problem regardless.

None of the lead maintainers responsible for formal releases of modules are POs.

All of the lead maintainers are developers who may not be involved in many of the changes in that are made to a module. 

This change is a good example of that, it is being done by a team separate from the lead maintainer and it is only chance that they (me) are aware of it.